### DISCUSSION TITLE
posts/C++%E5%9F%BA%E7%A1%80%E7%AC%94%E8%AE%B0/

### DISCUSSION THREAD

**Discussion by giscus** [Rainlv/rainlv.github.io#1] [Announcements]
# posts/C++%E5%9F%BA%E7%A1%80%E7%AC%94%E8%AE%B0/

静态变量

https://blog.rainna.xyz/posts/C++%E5%9F%BA%E7%A1%80%E7%AC%94%E8%AE%B0/

**Comment by Rainlv** [comment]
评论测试
